search/reverse_geocoder.cpp

```diff
@@ -77,6 +77,20 @@ string Join(string const & s, Args &&... args)
     return s;
   return s + ", " + tail;
 }
+
+ReverseGeocoder::Building FromFeatureImpl(FeatureType & ft, std::string hn, double distMeters)
```

Comment:
Тут не будет копии создаваться точно?
```suggestion
ReverseGeocoder::Building FromFeatureImpl(FeatureType & ft, std::string && hn, double distMeters)
```